brigade.js

```diff
@@ -42,6 +83,45 @@ function buildAndPublishImages(project, version) {
   return job;
 }
 
+// handleIssueComment handles an issue_comment event, parsing the comment text
+// and determining whether or not to trigger an action
+function handleIssueComment(e, p) {
+  payload = JSON.parse(e.payload);
+
+  // Extract the comment body and trim whitespace
+  comment = payload.body.comment.body.trim();
+
+  // Here we determine if a comment should provoke an action
+  switch(comment) {
+    case "/brig run":
+      runSuite(e, p);
```

Comment:
See pending PR #68; once that is merged, this should return the promise returned by `runSuite()`.

If we don't return a promise or throw an error, the caller of the handler will never know about any failures.

Reply:
👍 updated.